src/wp-admin/edit-comments.php

```diff
@@ -257,9 +281,9 @@
 		}
 
 		if ( $spammed > 0 ) {
-			$ids = isset($_REQUEST['ids']) ? $_REQUEST['ids'] : 0;
+			$ids = isset( $_REQUEST['ids'] ) ? $_REQUEST['ids'] : 0;
 			/* translators: %s: number of comments marked as spam */
-			$messages[] = sprintf( _n( '%s comment marked as spam.', '%s comments marked as spam.', $spammed ), $spammed ) . ' <a href="' . esc_url( wp_nonce_url( "edit-comments.php?doaction=undo&action=unspam&ids=$ids", "bulk-comments" ) ) . '">' . __('Undo') . '</a><br />';
+			$messages[] = sprintf( _n( '%s comment marked as spam.', '%s comments marked as spam.', $spammed ), $spammed ) . ' <a href="' . esc_url( wp_nonce_url( "edit-comments.php?doaction=undo&action=unspam&ids=$ids", 'bulk-comments' ) ) . '">' . __( 'Undo' ) . '</a><br />';
```

Comment:
Should this be expanded?

Reply:
We can see, it is a long line, matches upstream but it's a long concatenation - suggest for discussion.
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/edit-comments.php

Reply:
As per review meeting, this is fine to keep the change